Dockerfile.cuda

```diff
@@ -11,9 +14,10 @@ RUN mkdir -p /usr/local && tar xz -C /usr/local </tmp/go1.21.1.tar.gz
 COPY . .
 ENV GOARCH=$TARGETARCH
 RUN /usr/local/go/bin/go generate ./... \
+    && rm -f llm/llama.cpp/*/build/cuda/bin/*.so* \
```

Comment:
This trims unneeded shared libraries from the go embedded FS since it's available in the docker image